.gitignore

```diff
@@ -7,6 +7,7 @@ _build
 .project
 .pycharm_helpers/
 .pydevproject
+.vscode/
```

Comment:
If you put this in a global gitignore locally, you won't have to add it to individual repos.

Reply:
Ah ok, sounds good, I only use this editor for oeps but i will remove this, sorry